.github/workflows/bundle-stats.yml

```diff
@@ -0,0 +1,21 @@
+name: Bundle Stats
```

Comment:
Oh nice, will this mean we can remove bundlewatch? Where do we set the bundle size limits?

Reply:
I remember hearing that this check will fail until it's been merged into main, that could be the download artifacts error?

Reply:
Let's create an issue to remove bundlewatch once we verify that bundle stats are working as expected

Reply:
https://github.com/grafana/logs-drilldown/issues/1165

Reply:
Removed it for now and updated the ticket.